Core/src/OffConverter/OffReader.cpp

```diff
@@ -163,12 +164,10 @@ void OffReader::readTriangleFace(std::stringstream& lineStream,
 	buw::Vector3f normal = calcNormal(vector1, vector2, vector3);
 
 	//read color
-	buw::Vector3f color(0.0f, 0.0f, 1.0f); //default color; to be changed later on 
+	buw::Vector3f color = OffReader::readColorsFromFace(lineStream);
 
 	//create vertices with position, color and normal and add to list of all vertices
 	vertices.push_back(buw::VertexPosition3Color3Normal3(vector1, color, normal));
-	vertices.push_back(buw::VertexPosition3Color3Normal3(vector2, color, normal));
-	vertices.push_back(buw::VertexPosition3Color3Normal3(vector3, color, normal));
```

Comment:
Each vertex needs a color and a normal - this is probably the reason to see darker edges on the faces.

Reply:
I deleted it by mistake
Resolved in bbb7fb7